dev_modules/custom/health_sample_content/config/install/migrate_plus.migration.sample_node_listing_curated.yml

```diff
@@ -0,0 +1,129 @@
+id: sample_node_listing_curated
```

Comment:
@Tjocksmurf - given we don't have a Listing Curated content type it would be better to add the Listing - Curated to the sample standard page. This would also reduce the number of Percy images we need to generate and means we are less likely to exceed out free screenshot limit.